_includes/header.html

```diff
@@ -16,7 +16,7 @@
                 <li class="nav-item dropdown"> <a class="nav-link dropdown-toggle active" href="#"
                     data-toggle="dropdown">Home</a>
                   <ul class="dropdown-menu">
-                    <li><a class="dropdown-item" href="index.html">Landing 1</a>
+                    <li><a class="dropdown-item" href="contribution.html">Kontrybucja</a>
```

Comment:
Czy w naszym pięknym słowniku nie ma lepszego słowa niż kontrybucja? 😄 może 'dołącz do nas' czy 'udział'

ten komentarz dotyczy w sumie całego PRa

Reply:
Z wielką checią widziałbym coś innego :-) Mam takie same odczucia co do `kontrybucja` - brzmi trochę jakby pewne wydawnictwo coś tłumaczyło. `udział` ma w moim mniemaniu trochę negatywną konotację. Może za dużo filmów akcji albo kontaktu z prawem ;) `Dołącz do nas` brzmi dobrze :)

Reply:
Jeśli chcesz rzeczownik to masz "Uczestnictwo" ;)